src/components/ebay-skeleton/index.marko

```diff
@@ -0,0 +1,46 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+static var validTypeSizes = ["button", "text", "tile"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "form";
+$ var typeClass = `skeleton-${type}`;
+$ var skeletonClass = (
+    validTypeSizes.includes(input.type) &&
+    (size !== "default"
+        ? `${typeClass}--${size}`
+        : type === "text"
+        ? `${typeClass}--small`
+        : "")
+);
+
+<div
+    ...processHtmlAttributes(input, ignoredAttributes)
+    class=[
+        typeClass,
+        skeletonClass,
+        input.multiline && type === "text" && "skeleton-text--multiline",
+        input.class,
+    ]
+    role="img"
+    aria-label=input.a11yText || "Loading"
```

Comment:
Regarding point 1, IMO majority of eBay users use English and, adding a consistent a11y text makes it consistent experience rather than not passing at all. All our components provide similar consistent experience for non-English app teams to pass a localized value, so hopefully it makes easier to adopt. Point 2 is taken care of with current code refactor as we are always providing the wrapper and adding a11yText on the wrapper.